crates/forge/src/main.rs

```diff
@@ -108,6 +108,10 @@ struct TestArgs {
     /// Save execution traces of all test which have passed and are not fuzz tests
     #[arg(long)]
     save_trace_data: bool,
+
+    /// Max steps per test
+    #[arg(long)]
+    max_n_steps: Option<u32>,
```

Comment:
Can we have the same description as in docs? Clap parses this comment and it is shown in CLI if you type `snforge --help`